tests/e2e/operator/util/kubectl/kubectl.go

```diff
@@ -52,6 +59,46 @@ func kubectl(format string, args ...interface{}) string {
 	return binary + " " + fmt.Sprintf(format, args...)
 }
 
+// getConfig returns the configuration of the kubernetes go-client
+func getConfig() (*rest.Config, error) {
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+
+	// use the current context in kubeconfig
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		return nil, fmt.Errorf("error building config: %v", err)
+	}
+
+	return config, nil
+}
+
+// initClientset returns the kubernetes clientset and the apiextensions clientset
+func InitK8sClients() (*kubernetes.Clientset, *apiextensionsclient.Clientset, error) {
```

Comment:
This should probably be in a different file, not in `kubectl.go`, as it has nothing to do with `kubectl` itself.

Reply:
Yes, I put it here to save time and make all the test